src/views/Settings.vue

```diff
@@ -198,6 +247,50 @@ export default {
       this.currentPassword = "";
       this.newPassword = "";
       this.confirmNewPassword = "";
+    },
+    async shutdownPrompt() {
+      // Get user consent first
+      const approved = await this.$bvModal.msgBoxConfirm(
+        'Your Umbrel will be unresponsive untill you can get physical access to the device to power it back on.',
+        {title: "Are you sure?"}
+      );
+      if (!approved) {
+        return;
+      }
+
+      // TODO: Send shutdown request
+
+      this.$bvToast.toast(
+        `System is shutting down`,
+        {
+          title: "Your Umbrel will become unresponsive soon",
+          autoHideDelay: 3000,
+          variant: "danger",
+          solid: true,
+          toaster: "b-toaster-bottom-right"
+        }
+      );
+
+      // Hide the UI so they don't try to use it while Umbrel is down.
+      await delay(3000);
+      document.body.style.transition = 'opacity 2s ease'
```

Comment:
Also +1 for nicely fading out when the user requests shutdown. I think the [system store module](https://github.com/getumbrel/umbrel-dashboard/blob/master/src/store/modules/system.js) would be better handled for state management as well as the resulting fadeout. Will PR 👍 

We use [`vuex`](https://vuex.vuejs.org) for